server/src/com/thoughtworks/go/server/util/GoMacLauncher.java

```diff
@@ -41,76 +40,16 @@
     private Application application = Application.getApplication();
     private static final int MAX_SLEEP_SECONDS = 2400;
     private static final int SLEEP_MILLIS = 1000;
-    private static final String APPLICATION_SUPPORT_PATHNAME = System.getProperty("user.home")+"/Library/Application Support/Go Server/";
+    private static final String APPLICATION_SUPPORT_PATHNAME =
+            MessageFormat.format("{0}/Library/Application Support/{1}/", System.getProperty("user.home"), System.getProperty("go.application.name", "Go Server"));
     private static final String GO_CONFIG_DIRECTORY_PATH = APPLICATION_SUPPORT_PATHNAME + "config/";
     private static final String CRUISE_SERVER_URL = "http://localhost:8153/go/";
 
     public static void main(String[] args) {
         new File(GO_CONFIG_DIRECTORY_PATH).mkdirs();
-
-        //TODO: remove this call after a few releases
-        moveFilesFromGoServerDirToConfigDir();
```

Comment:
I think no one is (strike that, I mean, should be) using 12.2 (in "prod" on Mac) and considering an upgrade to 15.1. I hope no one is. I don't think we should spend time documenting that, unless someone does hit the issue. 12.2 is over three years old. Let me know, if you think that's ok. If not, I'll document it.

I feel we have too many of those in documentation ("Prior to 12.3.1, Go server stored its data in /Library/Application Support/Go Server. From 12.3.1, it will be in < user-home >/Library/Application Support/Go Server.") and it gets confusing.